lib/sass/script/value/callable.rb

```diff
@@ -0,0 +1,48 @@
+module Sass::Script::Value
+  # A SassScript object representing a null value.
+  class Callable < Base
+    # Constructs a Callable value for use in SassScript.
+    #
+    # @param callable [Sass::Callable] The callable to be used when the
+    # callable is called.
+    def initialize(callable)
+      super(callable.dup)
+    end
+
+    def to_s(opts = {})
+      raise Sass::SyntaxError.new("#{to_sass} isn't a valid CSS value.")
+    end
+
+    def inspect
+      to_sass
+    end
+
+    def to_sass
+      "#{value.type}-reference(#{value.name})"
```

Comment:
`value.name` here definitely should be quoted at least some of the time—what if it's a color name?